content/articles/eclat-algorithm-in-python/index.md

```diff
@@ -0,0 +1,148 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /eclat-algorithm-in-python/
+title: Getting Started with ECLAT Algorithm in Association Rule Mining
+description: This article will get you started with ECLAT Algorithm in Association Rule Mining.
+author: jackson-munyai
+date: 2021-12-20T00:00:00-21:00
+topics: [Machine Learning]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/eclat-algorithm-in-python/hero.png 
+    alt: Getting Started with ECLAT Algorithm Example Image
+---
+The goal in data mining is to find rules that predict an item based on the occurrence of other items in the transactions.
+<!--more-->
+
+Generally, to achieve this, we have two approaches;
+1. Generating the frequent itemsets
+Here, we generate all itemsets with greater support than the adopted minimum support.
+2. Rule Generation
+Generate a frequent high set from the frequent itemsets.
+
+### Prerequisites:
+To follow up with this article, the learner needs to be comfortable working with the following libraries in Python: Scikit-learn, Pandas, and Numpy.
+
+### Introduction to ECLAT Algorithm
+As we aforemebtioned, the main idea in the association rule is to discover valid information and knowledge from a large dataset. There are a number of algorithms that have been developed over the years that make this activity as successful as possible. The major algorithm used includes:
+
+1. Apriori Algorithm
+2. Eclat Algorithm
+3. FP Growth Algorithm
```

Comment:
@jacksonjate1 Since you've mentioned the `FP Growth Algorithm`, briefly talk about it.